test/test_prototype_transforms_consistency.py

```diff
@@ -1090,13 +1090,16 @@ def make_datapoints(self, with_mask=True):
         "t_ref, t, data_kwargs",
         [
             (det_transforms.RandomHorizontalFlip(p=1.0), v2_transforms.RandomHorizontalFlip(p=1.0), {}),
-            # FIXME: make
-            #  v2_transforms.Compose([
-            #      v2_transforms.RandomIoUCrop(),
-            #      v2_transforms.SanitizeBoundingBoxes()
-            #  ])
-            #  work
-            # (det_transforms.RandomIoUCrop(), v2_transforms.RandomIoUCrop(), {"with_mask": False}),
+            (
+                det_transforms.RandomIoUCrop(),
+                v2_transforms.Compose(
+                    [
+                        v2_transforms.RandomIoUCrop(),
+                        v2_transforms.SanitizeBoundingBoxes(labels_getter=lambda sample: sample[1]["labels"]),
+                    ]
+                ),
+                {"with_mask": False},
```

Comment:
do we still need this?

Reply:
Haven't looked into that one. Let me check.

Reply:
Yes we do. For some reason our references don't support the masks:

https://github.com/pytorch/vision/blob/3991ab994630ede87706dc9b46d005b8834cc5bc/references/detection/transforms.py#L143-L152